localstack-core/localstack/aws/gateway.py

```diff
@@ -25,3 +27,13 @@ def __init__(
             exception_handlers,
             context_class or RequestContext,
         )
+
+    def handle(self, context: RequestContext, response: Response) -> None:
+        """Exposes the same interface as ``HandlerChain.handle``."""
+        return self.new_chain().handle(context, response)
+
+    def process(self, request: Request, response: Response):
+        # TODO: this overrides might not be necessary
+        context = self.context_class(request)
+
+        self.handle(context, response)
```

Comment:
This looks identical to upstream?

Reply:
Yes, I think this was done earlier and was left in, but should be removed now. Good catch!